templates/MetadataGenerator/ANZSRC_Keywords.xml.twig

```diff
@@ -0,0 +1,34 @@
+<gmd:MD_Keywords>
+    {% if keywords is empty %}
```

Comment:
redundant. Question for PO: Is it alright to leave this entire section out if we don't have anzsrc keywords? (It's valid without it)

Reply:
Yes, leave it out.